test/e2e/scenario/recorder/shadowDom.scenario.ts

```diff
@@ -80,6 +82,34 @@ const divShadowDom = `<script>
  </script>
  `
 
+/** Will generate the following HTML 
+ * ```html
+ * <my-div id="titi">
+ *  #shadow-root
+ *    <div>toto</div>
+ *</my-div>
+ *```
+ when called like `<my-div />`
+ */
+const divWithStyleShadowDom = `<script>
+class DivWithStyle extends HTMLElement {
+ constructor() {
+   super();
+   this.attachShadow({ mode: "open" });
+ }
+ connectedCallback() {
+   const div = document.createElement("div");
+   div.textContent = 'toto'
+   this.shadowRoot.appendChild(div);
+   const styleSheet = new CSSStyleSheet();
+   styleSheet.insertRule('div { width: 100%; }')
+   this.shadowRoot.adoptedStyleSheets= [styleSheet]
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
   this.shadowRoot.adoptedStyleSheets = [styleSheet]
```